web/src/util/auth.js

```diff
@@ -158,3 +161,25 @@ export const isUserActive = latestActivity => {
   const now = new Date().getTime();
   return now - latestActivity < INACTIVITY_LIMIT;
 };
+
+// Cookie
+
+const cookieName = 'gov.cms.eapd.hasConsented';
+
+export const cookie = name => {
+  const cookieMap = (document.cookie || '').split(';').reduce((c, s) => {
+    const bits = s.trim().split('=');
+    if (bits.length === 2) {
+      return { ...c, [bits[0].trim()]: bits[1].trim() };
+    }
+    return c;
+  }, {});
+
+  return cookieMap[name];
+};
```

Comment:
I'm curious why we have implementing our own 'cookie string to object' code in this app. It might be fine, but we should weight the pros and cons of using an external library to do this work. 

https://openbase.com/categories/js/best-nodejs-cookies-libraries?orderBy=RECOMMENDED&